compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -386,8 +387,6 @@ static void build_chpl_entry_points() {
   if (fLibraryCompile) {
     if (chpl_user_main)
       INT_FATAL(chpl_user_main, "'main' found when compiling a library");
-    if (mainModules.n != 1)
```

Comment:
I agree -- if/when we have the restriction, something like this might need to be done; but at present I couldn't see any reason that the limit should be there and it was hard to preserve the check simply/cleanly (a check that was inherited from the days when we compiled part of the runtime with the compiler and there really did have to be just one module).  

Even if, in the future, there were a "root" module in the library, it seems more related to the --main-module concept we have at present rather than restricting that the library may only contain one module.

[@safl and @lydia-duncan may want to note the relaxing of this restriction as it could have an impact (maybe) on PyChapel work (in that multi-module libraries could be created -- assuming this is the mode that PyChapel uses).]